### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by carlopi**
Couple of minor fixes, mostly adds some context around storage mismatches (that made sense to me, but is anyone wants to iterate it would be welcome)

### CODE REVIEW

File: _includes/installation.html [hunk 1]
```diff
@@ -136,7 +136,24 @@ <h3>Usage Example</h3>
...
 
 	<div class="latest cli source">
-		coming soon
+		git clone https://github.com/duckdb/duckdb.git
+		<br/>
+		cd duckdb
+		<br/>
+		git checkout v{{ site.currentduckdbversion }}
+		<br/>
+		git submodule init --update
```
Comment (Mause) [inline _includes/installation.html:145]: I don't think we use any submodules in the main repo?

File: internals/storage.md [hunk 2]
```diff
@@ -5,7 +5,37 @@ title: Storage
 
 The DuckDB internal storage format is currently in flux, and is expected to change with each release until we reach v1.0.0.
 
+## How to move between storage formats
+
+When you update duckdb and open a database file, you might encounter an error message about incompatible storage formats, pointing to this page.
+To move your database(s) to newer format you only need the older and the newer duckdb executable.
+
+Opening your database file with older duckdb and usign the SQL statement `"EXPORT DATABASE 'tmp';"` allows to save the whole state of the current database in use inside folder `tmp`.
```
Comment (Mause) [inline internals/storage.md:13]:
```suggestion
Opening your database file with the older duckdb and using the SQL statement `"EXPORT DATABASE 'tmp';"` allows you to save the whole state of the current database in use inside folder `tmp`.
```

File: internals/storage.md [hunk 3]
```diff
@@ -5,7 +5,37 @@ title: Storage
...
+`tmp` content will be overridden, so choose an empty/non yet existing folder.
+
+Starting then the newer duckdb and executing `"IMPORT DATABASE 'tmp';"` (pointing to the previously populated folder) allow to load the database, that can be then saved to the file you pointed duckdb to.
```
Comment (Mause) [inline internals/storage.md:16]:
```suggestion
Then start the newer duckdb and execute `"IMPORT DATABASE 'tmp';"` (pointing to the previously populated folder) allows you to load the database, which can be then saved to the file you pointed duckdb to.
```

File: internals/storage.md [hunk 4]
```diff
@@ -5,7 +5,37 @@ title: Storage
...
+
+A bash two-liner (to be adapted with the right file names and executable locations) is:
+```bash
+user@device $ /older/version/duckddb mydata.db -c "EXPORT DATABASE 'tmp';"
+user@device $ /newer/duckdb mydata.new.db -c "IMPORT DATABASE 'tmp';"
```
Comment (Mause) [inline internals/storage.md:21]:
```suggestion
$ /older/version/duckdb mydata.db -c "EXPORT DATABASE 'tmp';"
$ /newer/duckdb mydata.new.db -c "IMPORT DATABASE 'tmp';"
```

File: internals/storage.md [hunk 5]
```diff
@@ -5,7 +5,37 @@ title: Storage
...
+After this `mydata.db` will be untouched with the old format, `mydata.new.db` will contain the same data but in a format accessible from more recent duckdb, and folder `tmp` will old the same data in an universal format as different files.
+
+Check [EXPORT documentation](docs/sql/statements/export) for more details on the syntax.
+
+## Storage header
+
 DuckDB files start with a `uint64_t` which contains a checksum for the main header, followed by four magic bytes (`DUCK`), followed by the storage version number in a `uint64_t`.
+
+```bash
+user@device $ hexdump -n 20 -C mydata.db
```
Comment (Mause) [inline internals/storage.md:33]:
```suggestion
$ hexdump -n 20 -C mydata.db
```

**Review by Mause — CHANGES_REQUESTED** [review]
Just a couple of small types :duck:

**Comment by carlopi** [comment]
@Mause: Thanks for the feedback!

**Comment by Mause** [comment]
Thanks!

**Pull request merged**